src/terminal/terminal.rs

```diff
@@ -568,6 +568,65 @@ where
     }
 }
 
+#[cfg(feature = "crossterm")]
+impl Terminal<CrosstermBackend<Stdout>> {
+    /// Initializes the terminal with reasonable defaults for most applications.
+    ///
+    /// The Terminal is initialized with the following settings:
+    ///
+    /// - Stdout as the output
+    /// - Crossterm backend
+    /// - Raw mode enabled
+    /// - Alternate screen buffer enabled
+    /// - A panic hook that restores the terminal before panicking
+    ///
+    /// For more control over the terminal initialization, use [`Terminal::new`] or
+    /// [`Terminal::with_options`].
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// let terminal = ratatui::Terminal::init()?;
+    /// # std::io::Result::Ok(())
+    /// ```
```

Comment:
Should the docstring example use `color_eyre`?

```suggestion
    /// ```rust,no_run
    /// color_eyre::install()?;
    /// let terminal = ratatui::Terminal::init()?;
    /// # std::io::Result::Ok(())
    /// ```
```

Reply:
If this is a reasonably acceptable approach, I'd do a pass over the entire crate and fixup examples. Probably not worth adding to the example here however, but it's worth noting some info about anyhow / color-eyre in the doc comments.